appsembler_api/views.py

```diff
@@ -449,11 +447,10 @@ class EnrollUserWithEnrollmentCodeView(APIView):
     permission_classes = (IsStaffOrOwner,)
 
     def post(self, request):
-        # TODO: this is also affected by the removal of commerce
-        raise NotImplementedError
+        if is_ratelimited(request, key="user", group="enrollment-codes.enroll-user", rate="6/m"):
```

Comment:
BadRequestRateLimiter was removed in edx-platform, so I switched to using the django-ratelimit library, which I found is one currently used and [recommended in edx-platform](https://github.com/openedx/edx-platform/blob/c27edbc57819392f88c57b4b075f2c6e17d4624b/docs/decisions/0009_simplify_ratelimiting.rst?plain=1#L17).